Makefile.in

```diff
@@ -116,15 +117,19 @@ java/libjlog.jnilib:	java/com_omniti_labs_jlog.lo $(SOOBJS)
 install:
 	$(srcdir)/mkinstalldirs $(DESTDIR)$(bindir)
 	$(srcdir)/mkinstalldirs $(DESTDIR)$(libdir)
+	$(srcdir)/mkinstalldirs $(DESTDIR)$(libdir)/java
 	$(srcdir)/mkinstalldirs $(DESTDIR)$(includedir)
 	$(INSTALL) -m 0755 jlogctl $(DESTDIR)$(bindir)/jlogctl
 	$(INSTALL) -m 0755 jlogtail $(DESTDIR)$(bindir)/jlogtail
 	$(INSTALL) -m 0755 jlog_change_endian.pl $(DESTDIR)$(bindir)/jlog_change_endian
 	$(INSTALL) -m 0755 jlog_sanity_check.pl $(DESTDIR)$(bindir)/jlog_sanity_check
-	$(INSTALL) -m 0755 libjlog.a $(DESTDIR)$(libdir)/libjlog.a
-	$(INSTALL) -m 0755 libjlog.$(DOTSO) $(DESTDIR)$(libdir)/$(LIBLONG)
-	$(LN_S) -f $(LIBLONG) $(DESTDIR)$(libdir)/$(LIBSHORT)
-	$(LN_S) -f $(LIBLONG) $(DESTDIR)$(libdir)/$(LIBMAJOR)
+	$(INSTALL) -m 0755 libjlog.a $(DESTDIR)$(libdir)/java/libjlog.a
+	$(INSTALL) -m 0644 java/jlog.jar $(DESTDIR)$(libdir)/java/jlog.jar
+	$(INSTALL) -m 0755 java/libjlog.jnilib $(DESTDIR)$(libdir)/java/libjlog.jnilib
+	$(LN_S) -f $(DESTDIR)$(libdir)/java/libjlog.jnilib $(DESTDIR)$(libdir)/java/libjnijlog.so
+	$(INSTALL) -m 0755 libjlog.$(DOTSO) $(DESTDIR)$(libdir)/java/$(LIBLONG)
+	$(LN_S) -f $(LIBLONG) $(DESTDIR)$(libdir)/java/$(LIBSHORT)
+	$(LN_S) -f $(LIBLONG) $(DESTDIR)$(libdir)/java/$(LIBMAJOR)
```

Comment:
these three are bad too.. libjlog.so goes in the libdir, it has nothing to do with Java.